src/config/parameters.php

```
@@ -1,3 +1,20 @@
 <?php
 /** @var Symfony\Component\DependencyInjection\Loader\Configurator\ParametersConfigurator $parameters */
-$parameters->set('env', 'dev');
\ No newline at end of file
+
+$parameters->set('env', 'dev');
+if (file_exists(".env.dev")) {
+    $env = file(".env.dev");
+} else if (file_exists(".env")) {
+    $env = file(".env");
+} else {
+    var_dump("there are no parameters to start the server");
+    die;
```

you need to throw an exception